core/fuzz/utils.rs

```diff
@@ -20,6 +20,19 @@ use std::env;
 use opendal::Operator;
 use opendal::Scheme;
 
+pub fn apply_args_to_env() {
```

Comment:
Why we need this?
The process method in oss-fuzz will make a .options file like
```
[libfuzzer]
OPENDAL_MEMORY_TEST=on
```
to `-opendal_memory_test=on` and append it into program agrs.

```
/out/fuzz_reader_fs -rss_limit_mb=2560 -timeout=25 /tmp/fuzz_reader_fs_corpus -opendal_memory_test=on
```

Reply:
I got it and I don't like it.